MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -875,6 +875,15 @@
 	_metalFeatures.minSwapchainImageCount = kMVKMinSwapchainImageCount;
 	_metalFeatures.maxSwapchainImageCount = kMVKMaxSwapchainImageCount;
 
+#if MVK_TVOS
+	_metalFeatures.mslVersionEnum = MTLLanguageVersion1_0;
+    _metalFeatures.maxPerStageTextureCount = 31;
+    _metalFeatures.mtlBufferAlignment = 64;
+	_metalFeatures.mtlCopyBufferAlignment = 1;
+    _metalFeatures.texelBuffers = true;
+	_metalFeatures.maxTextureDimension = (4 * KIBI);
```

Comment:
Features need to be defined here for tvOS. Where can I find the list of supported features per tvOS gpu family?

Reply:
You can see the features for the new Metal 3.0 scheme [here](https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf). I have an older version of that document that lists features supported by the old feature set scheme, but I can't upload it because GitHub uploads are broken.

Reply:
Thanks! Do I need the old doc, or is the new one sufficient. If uploads aren't working, could you email to aman@tmm1.net

Reply:
The new one only lists by GPU family; those were only introduced in Metal 3.0. For Metal 2.2 and below (i.e. any version of tvOS prior to 13.0), you need the old feature set scheme, so you know which feature set enums to use to gate certain features. I'll send you that PDF.

Reply:
Appreciate it!

Looks like in general:

>iOS GPU family 2 == tvOS GPU family 1
>iOS GPU family 3 == tvOS GPU family 2

So I can copy over the `MVK_IOS` entries and rename accordingly.

Reply:
[Metal-Feature-Set-Tables.pdf](https://github.com/KhronosGroup/MoltenVK/files/4789658/Metal-Feature-Set-Tables.pdf)